docs/cosmology/traits.rst

```diff
@@ -0,0 +1,54 @@
+.. _astropy-cosmology-traits:
+
+****************
+Cosmology Traits
+****************
+
+.. currentmodule:: astropy.cosmology.traits
+
+The ``traits`` module hosts various parts of cosmologies, such as the
```

Comment:
```suggestion
The ``traits`` module hosts various parts of cosmologies, such as the
```
Add a link to the module documentation.

Reply:
?

Reply:
Turning ``traits`` into a link makes the HTML documentation look better.